src/Microsoft.AspNetCore.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -20,18 +20,11 @@ public abstract class AuthenticationManager
 
         public abstract IEnumerable<AuthenticationDescription> GetAuthenticationSchemes();
 
-        public abstract Task AuthenticateAsync(AuthenticateContext context);
+        public abstract Task<AuthenticateInfo> GetAuthenticateInfoAsync(string authenticationScheme);
```

Comment:
Well, TBH, how many people are using passive authentication these days? 5%? 10%?
These APIs are rather complicated for beginners and most will prefer using active authentication (which is the default for both cookies and bearer) or use Identity. In this case, retrieving the principal via `HttpContext.User` or `ControllerBase.User` will be much easier.